server/i18n/en.json

```diff
@@ -231,6 +231,22 @@
     "id": "api.channel.add_user_to_channel.type.app_error",
     "translation": "Can not add user to this channel type."
   },
+  {
+    "id": "api.channel.bookmark.create_channel_bookmark.forbidden.app_error",
+    "translation": "Failed to create the channel bookmark."
+  },
+  {
+    "id": "api.channel.bookmark.create_channel_bookmark.license.error",
+    "translation": "Your license does not support channel bookmarks."
```

Comment:
@esethna can you clarify if this is a EE feature? not to my recollection.

Reply:
This is based on the prototypes of the system console permissions that reference the feature being enterprise only

Reply:
ok let's wait for @esethna to confirm, if it is, we should move all the logic to the enterprise repo.